gremlin-go/docker/gremlin-server-integration-secure.yaml

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
we had managed to reduce the number of copies of these gremlin server config files down to:

* the template one in `gremlin-server/src/test/resources/org/apache/tinkerpp/gremlin/server` - this one is dynamically reconfigured by the groovy server start scripts to produce the three server variations that are tested
* the three configuration files that statically represent the three server variations described above and are stored in `docker/gremlin-server/`

How do we avoid duplicating these here in gremlin-go? and, if gremlin-go is meant to be the model for other GLVs then i think it would be worth including the third configuration even if we dont' support kerberos here.